### PR TITLE
Update documentation links

### LINKED ISSUE #5775: Update links in documentation to match new site urls
We've migrated materials from cesiumjs.org to cesium.com. Some links in our documentation rely on a redirect to the new location, but we should update all links pointing to cesiumjs.org to the new valid locations on cesiumjs.org or cesium.com.

### PR DESCRIPTION
**Pull request by hanbollar**
Fixes #5775

Updated documentation links and changed https to http

### CODE REVIEW

**Comment by cesium-concierge** [comment]
[Please sign the CLA before we review this PR.](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla)

Welcome to the Cesium community @hanbollar!

Can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
Thanks @hanbollar!  Also add yourself to [CONTRIBUTORSmd](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTORS.md) under the AGI section

File: CHANGES.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Change Log
    * Only one node is supported.
    * Only one mesh per node is supported.
    * Only one primitive per mesh is supported.
+* Updated documentation links to reflect new locations on cesium.com.
```
Comment (hpinkos) [inline CHANGES.md]: They're on `cesiumjs.org`, not `cesium.com`

Comment (hpinkos) [reply CHANGES.md]: Oh , nevermind, I see you're referring to some of the blog links, right?

**Comment by hanbollar** [comment]
thanks! @hpinkos updated

**Comment by ggetz** [comment]
We have the updated the `CLA.json` to include @hanbollar.

**Comment by mramato** [comment]
Welcome to the team!

**Comment by pjcozzi** [comment]
Thanks @hanbollar, congrats on your first Cesium PR!

**Pull request merged**